OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj

```diff
@@ -37,10 +37,19 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <Content Remove="stylecop.json" />
+        <Content Remove="wwwroot\js\logout.js" />
```

Comment:
we already have watch on the wwwroot folder, please remove this

Reply:
Was removed